src/devices/video/virge_pci.cpp

```diff
@@ -95,7 +99,8 @@ uint32_t virge_pci_device::base_address_r()
 void virge_pci_device::base_address_w(offs_t offset, uint32_t data)
 {
 	pci_device::address_base_w(offset,data);
-	downcast<s3virge_vga_device *>(m_vga.target())->set_linear_address(data & 0xffff0000);
+	// only bits 31-26 are changed here, cfr. page 25-4
+	downcast<s3virge_vga_device *>(m_vga.target())->set_linear_address(data & 0xfc000000);
```

Comment:
Pretty sure this should really RMW whatever is in get_linear_address bits 25-0, win98se doesn't use it but other stuff may.